addons/test_mail/tests/test_message_track.py

```diff
@@ -353,6 +359,9 @@ def test_message_track_monetary(self):
             ('company_currency', 'many2one', self.user_employee.company_id.currency_id, self.company_2.currency_id)
         ])
 
+    def test_message_track_monetary_no_company(self):
+        self.record_no_company.write({'new_value_monetary': 300})
```

Comment:
You should implement try... except... to catch failure instead to let it crash
```suggestion
        try:
            self.record_no_company.write({'new_value_monetary': 300})
        except AttributeError:
            self.fail('AttributeError should not be raised when modifying tracked monetary fields')
```

Reply:
> 

I understood the problem